packages/shared/locales/en.json

```diff
@@ -655,6 +655,24 @@
                 "title": "Vesting of tokens",
                 "body": "From the tokens users received through an IOTA token airdrop, 10% are being available to users immediately at the genesis of the IOTA Stardust network. The rest is gradually unlocked over the course of two years in equal proportion happening every two weeks."
             }
+        },
+        "implicit-account":{
+            "title": "Activate Account",
+            "steps" : {
+                "step1": {
```

Comment:
I think that by following the same dynamics as with the rest of the cases in which there are `steps`, it should be called `"0"`. `step1` -> `0`
as in `ledgerAppGuide` and `ledgerConnectionGuide`